js/views/itemEditVw.js

```diff
@@ -663,5 +663,10 @@ module.exports = baseVw.extend({
     this.validateDescription();
 
     return this.$('#contractForm')[0].checkValidity();
+  },
```

Comment:
There's a clone() of a model happening in renderItemEdit... maybe that has something to do with the data not being cleared?